catalog/bucket/bucket.yaml

```diff
@@ -14,14 +14,13 @@
 apiVersion: storage.cnrm.cloud.google.com/v1beta1
 kind: StorageBucket
 metadata:
-  name: blueprints-project-bucket # kpt-set: ${project-id}-${name}
-  namespace: config-control
+  name: PLEASE_UPDATE
```

Comment:
This isn't a pattern I want to follow. We should still have semantically useful example values in place, not just "PLEASE_UPDATE."

Semantically useful values are important because they:
1. Can sometimes be used directly (ex. a location of `us-central1` can work fine)
2. Hint at the expected input type/format.

Reply:
Thank you! Yes, Brian pointed that out as well. Here's the changes I'll make
1. `metatada.name` uses the blueprint name
2. required field uses "PLACEHOLDER"
3. location/zone/region use a valid value.

Reply:
Where possible I would like to avoid PLACEHOLDER values entirely. We can provide a sensible example value in most cases.